src/core.c/Exception.pm6

```diff
@@ -1037,33 +1037,33 @@ my class X::Worry::Precedence::Range is X::Worry {
     }
 }
 
-my class X::Trait::Invalid is Exception {
+my role X::Trait is Exception {
     has $.type;       # is, will, of etc.
     has $.subtype;    # wrong subtype being tried
-    has $.declaring;  # variable, sub, parameter, etc.
-    has $.name;       # '$foo', '@bar', etc.
+    has $.declaring;  # variable, sub, parameter, etc. (optional)
+}
+my class X::Trait::Invalid does X::Trait {
+    has $.name;       # target of trait, e.g., '$foo' in `$foo is rw`
```

Comment:
“the the”

Reply:
Pardon? I don't see a duplicated "the".  Or maybe I don't understand?

Reply:
Oh, this got put one line up. Below, “reason the the trait”.

Reply:
Oh, I thought I checked the surrounding lines.  I guess not [well enough](https://en.wikipedia.org/wiki/Paris_in_the_the_Spring)! Thanks; fixed.